### PR TITLE
add support for importing aws_ssm_maintenance_window_target resource

### LINKED ISSUE #12910: Support aws_ssm_maintenance_window_target Resource Import
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

The `aws_ssm_maintenance_window_target` resource does not currently support import. It would be great if it did!

Implementation should include:
- [x] Resource `Importer` `State` implementation
- [x] Acceptance testing with `ImportState` and `ImportStateVerify`
- [x] Documentation

### Affected Resource(s)

* aws_ssm_maintenance_window_target

### References

* [Contributing Guide section on Adding Resource Import Support](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#adding-resource-import-support)

### PR DESCRIPTION
**Pull request by anGie44**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Closes #12910 

Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note
add support for importing aws_ssm_maintenance_window_target resource
```

Output from acceptance testing:

```
--- PASS: TestAccAWSSSMMaintenanceWindowTarget_validation (2.48s)
--- PASS: TestAccAWSSSMMaintenanceWindowTarget_disappears (12.73s)
--- PASS: TestAccAWSSSMMaintenanceWindowTarget_noNameOrDescription (13.92s)
--- PASS: TestAccAWSSSMMaintenanceWindowTarget_resourceGroup (14.07s)
--- PASS: TestAccAWSSSMMaintenanceWindowTarget_basic (14.45s)
--- PASS: TestAccAWSSSMMaintenanceWindowTarget_update (24.63s)
```

### CODE REVIEW

**Comment by anGie44** [comment]
@bflad i'll make changes to bring back the Checks as discussed in https://github.com/terraform-providers/terraform-provider-aws/pull/12921#pullrequestreview-398474458

File: aws/resource_aws_ssm_maintenance_window_target.go [hunk 1]
```diff
@@ -17,6 +17,17 @@ func resourceAwsSsmMaintenanceWindowTarget() *schema.Resource {
 		Read:   resourceAwsSsmMaintenanceWindowTargetRead,
 		Update: resourceAwsSsmMaintenanceWindowTargetUpdate,
 		Delete: resourceAwsSsmMaintenanceWindowTargetDelete,
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				idParts := strings.Split(d.Id(), ":")
+				if len(idParts) != 2 || idParts[0] == "" || idParts[1] == "" {
+					return nil, fmt.Errorf("Unexpected format of ID (%q), expected WINDOW_TARGET_ID:WINDOW_ID", d.Id())
```
Comment (bflad) [inline aws/resource_aws_ssm_maintenance_window_target.go:24]: Let's update this to look like [`aws_ssm_maintenance_window_task`'s import](https://www.terraform.io/docs/providers/aws/r/ssm_maintenance_window_task.html#import) ( WINDOW_ID/WINDOW_TARGET_ID) so operators have a similar experience between these two resources. 😄 

For import IDs where one resource is containing another, we do try to put container, delimiter, contained. 👍

**Comment by anGie44** [comment]
@bflad note latest commit :)

File: website/docs/r/ssm_maintenance_window_target.html.markdown [hunk 2]
```diff
@@ -73,3 +73,11 @@ The following arguments are supported:
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - The ID of the maintenance window target.
+
+## Import
+
+SSM Maintenance Window targets can be imported using `WINDOW_TARGET_ID:WINDOW_ID`, e.g.
+
+```
+$ terraform import aws_ssm_maintenance_window_target.example 23639a0b-ddbc-4bca-9e72-78d96EXAMPLE:mw-0c50858d01EXAMPLE
```
Comment (bflad) [inline website/docs/r/ssm_maintenance_window_target.html.markdown:82]: Needs to be updated to the new format: 😄 

``````suggestion
SSM Maintenance Window targets can be imported using `WINDOW_ID/WINDOW_TARGET_ID`, e.g.

```
$ terraform import aws_ssm_maintenance_window_target.example mw-0c50858d01EXAMPLE/23639a0b-ddbc-4bca-9e72-78d96EXAMPLE
``````

Comment (anGie44) [reply website/docs/r/ssm_maintenance_window_target.html.markdown:82]: ahh totally missed this file! 🙏

**Pull request merged**